pkg/dao/repository_configs.go

```diff
@@ -385,15 +385,25 @@ func (r repositoryConfigDaoImpl) List(
 	var totalRepos int64
 	repoConfigs := make([]models.RepositoryConfiguration, 0)
 	var contentPath string
+	var filteredDB *gorm.DB
 
-	accessibleFeatures, err := r.fsClient.GetEntitledFeatures(ctx, OrgID)
-	if err != nil {
-		return api.RepositoryCollectionResponse{}, totalRepos, err
-	}
+	if OrgID == config.RedHatOrg {
+		// Bypass the feature service status API if listing repos with the RH org
```

Comment:
would do you think about doing this check from the `GetEntitledFeatures` call function itself? This way we don't have to remember to check this in other places if we're using the red hat org

Reply:
ah yep that's a much better idea 😄 just pushed that up